debian/netscanner.manpage

```diff
@@ -0,0 +1,15 @@
+.\" Manpage for netscanner.
+.\" Contact dominic.orsi@gmail.com to correct errors or typos.
+.TH man 1 "22 June 2024" "1.0" "netscanner man page"
+.SH NAME
+netscanner \- view and scan connected networks
+.SH SYNOPSIS
+netscanner
+.SH DESCRIPTION
+Netscanner is a terminal network scanner & diagnostic tool with modern TUI.
+.SH OPTIONS
+Netscanner does not take any options.
```

Comment:
Calling `netscanner -h` will show you actual options. For now only tick & frame rates could be passed, but I will be adding another one in next version. All of it will be described in the README.md.

Reply:
Updated with commands, see commit [5a3019](https://github.com/Chleba/netscanner/pull/33/commits/5a301395f3004340d61affffcb9fb3c3f8d3cb66)